.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Check whether minirust succeeds to build.
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: download minirust-tooling
+        run: |
+          git clone "https://github.com/memoryleak47/minirust-tooling" ../minirust-tooling
+          cd ../minirust-tooling
+          git checkout 19f3392 # Fixed commit, we need to bump this occasionally.
```

Comment:
This will probably be part a config file at some point.

Reply:
I think instead of having a ton of tiny actions with shell scripts in them, I'd prefer one action that just calls `bash ci.sh` and then that script does the rest.

Reply:
Are you fine with having just one large action?
I kinda fear someone might actually run `ci.sh` on their computer.

Reply:
I have such `ci.sh` scripts in most of my projects and never had people run into problems. I think it's better to keep shell script logic outside these awfully verbose yml files with their strange escaping and indentation rules.

Reply:
@RalfJung alright, should be ready to review now